src/components/Block/Block.tsx

```diff
@@ -119,12 +120,20 @@ export default observer(function Block(props: BlockProps) {
     }
 
     return (
+        <>
+         { (blocks.groupBy && block.visible) ? <> {
+                block.groupName && 
+                    <span className='ReactTimeline__Block-GroupLabel' style={{left: `${ spaces.timeToPx(block.timespan.start)}px`, top: `${block.y - viewport.top - 25}px`, position: 'absolute'}}> 
+                        { (block.groupName !== 'nan') && block.groupName}
```

Comment:
I think this should be
```suggestion
                        { (block.groupName.toLowerCase() !== 'nan') && block.groupName}
```